test/e2e/util.go

```diff
@@ -161,6 +161,10 @@ type TestContextType struct {
 	GatherMetricsAfterTest            bool
 	// Currently supported values are 'hr' for human-readable and 'json'. It's a comma separated list.
 	OutputPrintType string
+	// CreateTestingNS is responsible for creating namespace used for executing e2e tests.
+	// It accepts namespace base name, which will be prepended with e2e prefix, kube client
+	// and labels to be applied to a namespace.
+	CreateTestingNS func(baseName string, c *client.Client, labels map[string]string) (*api.Namespace, error)
```

Comment:
I'm also still a little confused by the plumbing here - it seems like this is *only* used by the framework `beforeEach`, but not tests that call `CreateTestingNS` outside of framework. Is that intentional? Or should the `CreateTestingNS` func here in util.go DTRT and either delegate to `testContext.CreateTestingNS` or run itself?